### PR TITLE
Pioneer 18a

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
image

**Comment by cdobs** [comment]
image

**Comment by cdobs** [comment]
AR67

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -279,6 +279,8 @@ EK20220415,Elakha,2022-04-15T14:40,2022-04-15T23:30,CSPP deployment
 SU20220427,Pacific Surveyor,2022-04-27T12:46:00,2022-04-28T00:51:00,Endurance glider and CSPP ops
 SU20220428,Pacific Surveyor,2022-04-28T12:58:00,2022-04-28T21:27:00,Endurance CSPP ops
 EK20220428,Elakha,2022-04-28T16:11:00,2022-04-28T19:00:00,Endurance glider recovery
+AR67,Neil Armstrong,2022-05-06T00:00:00,2022-06-07T23:59:00,Pioneer glider cp_339 recovery
 EK20220510,Elakha,2022-05-10T16:30:00,2022-05-10T22:04:00,Endurance glider and CSPP deployments
 SKQ202208S,Sikuliaq,2022-05-11T00:00:00,2022-05-30T23:59:00,Papa 9
-AR69-01,Neil Armstrong,2022-06-20T00:00:00,2022-07-17T23:59:00,Irminger 9
+TS20220518,Tenacious II,2022-05-18T11:00:00,2022-05-19T03:00:00,Pioneer 18a glider deployments
+AR69-01,Neil Armstrong,2022-06-20T00:00:00,2022-07-17T23:59:00,Irminger 9
```
Comment (snwhite) [inline cruise/CruiseInformation.csv:286]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**